zircon.jvm/src/test/kotlin/org/hexworks/zircon/internal/component/impl/DefaultLogAreaTest.kt

```diff
@@ -0,0 +1,100 @@
+package org.hexworks.zircon.internal.component.impl
+
+import org.assertj.core.api.Assertions.assertThat
+import org.hexworks.zircon.api.Components
+import org.hexworks.zircon.api.builder.component.ComponentStyleSetBuilder
+import org.hexworks.zircon.api.builder.component.LogAreaBuilder
+import org.hexworks.zircon.api.builder.graphics.StyleSetBuilder
+import org.hexworks.zircon.api.color.TileColor
+import org.hexworks.zircon.api.data.Position
+import org.hexworks.zircon.api.data.Size
+import org.hexworks.zircon.api.resource.BuiltInCP437TilesetResource
+import org.hexworks.zircon.api.resource.ColorThemeResource
+import org.hexworks.zircon.api.resource.TilesetResource
+import org.hexworks.zircon.internal.component.impl.log.DefaultLogArea
+import org.hexworks.zircon.internal.component.impl.log.LogComponentElement
+import org.hexworks.zircon.internal.component.impl.log.LogTextElement
+import org.junit.Before
+import org.junit.Test
+
+class DefaultLogAreaTest {
+
+    lateinit var target: DefaultLogArea
```

Comment:
I really like that you took the naming conventions into consideration here (and for the fact that you tested the component). 👍